### PR TITLE
🐳 Adds Docker and Moonpay

### PR DESCRIPTION
**Pull request by nunocoracao**
* Adds Docker and Moonpay
* Fixed SingleStore position

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,3 +404,5 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [Avenue](https://avenue.place) [:rocket:](https://careers.avenue.place) | A communications and collaboration platform for DAOs. | `Porto` `Remote` |
+| [Achorage Digital](https://www.anchorage.com) [:rocket:](https://www.anchorage.com/careers) | Trusted institutional partner in crypto and first federally chartered crypto bank. | `Porto` `Remote` |
```
Comment (vpereira01) [inline README.md:407]: Anchorage is already listed, please update the existing one or remove this change.

File: README.md [hunk 2]
```diff
@@ -402,3 +404,5 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
...
+| [Moonpay](https://www.moonpay.com) [:rocket:](https://www.moonpay.com/careers) | End-to-end solutions for payments, enterprise-scale smart contract development, and digital asset management. | `Lisboa` `Remote` |
```
Comment (vpereira01) [inline README.md:408]: Please move this into the FinTech category and reduce the description to less than 70 chars per guidelines https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md?plain=1#L26

File: README.md [hunk 3]
```diff
@@ -180,6 +180,8 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Rows](https://www.rows.com) [:rocket:](https://github.com/dashdash/hiring) | Create interactive web apps using spreadsheet skills. | `Porto` |
 | [Synopsys](https://www.synopsys.com) [:rocket:](https://synopsys.ongig.com/?l=Portugal&regions=EMEA) | Silicon chip design, verification, IP integration and application security | `Lisbon` `Porto` |
 | [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) [:octocat:](https://github.com/Unbabel) | Translation as a Service. | `Lisboa` |
+| [Docker](https://docker.com) [:rocket:](https://www.docker.com/careers/) [:octocat:](https://github.com/docker) | Docker helps developers bring their ideas to life by conquering the complexity of app development. | `Remote` |
```
Comment (vpereira01) [inline README.md:183]: Please place this in alphabetical order, ie after Datadog, and short the description to be less than 70 chars per guidelines https://github.com/marmelo/tech-companies-in-portugal/blob/master/CONTRIBUTING.md

**Review by vpereira01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nunocoracao) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (nunocoracao) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (nunocoracao) [reply README.md]: fixed, some of the items are not in alphabetical order

File: README.md [hunk 4]
```diff
@@ -173,15 +173,17 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
...
 | [OutSystems](https://www.outsystems.com) [:rocket:](https://www.outsystems.com/company/careers/) | Low-code platform to visually develop applications. | `Braga` `Linda-a-Velha` <br> `Proença-a-Nova` |
 | [Reviewpad](https://www.reviewpad.com) [:rocket:](https://reviewpad.com/discord/hiring) [:octocat:](https://github.com/reviewpad) | Pull request automations to safely merge faster with happy developers. | `Porto` <br> `Remote` |
 | [Rows](https://www.rows.com) [:rocket:](https://github.com/dashdash/hiring) | Create interactive web apps using spreadsheet skills. | `Porto` |
+| [SingleStore](https://www.singlestore.com/) [:rocket:](https://www.singlestore.com/careers/jobs/) | The scalable cloud database for data-intensive applications, deployed on AWS, Azure, or GCP. (formerly MemSQL) | `Lisboa` `Remote` |
 | [Synopsys](https://www.synopsys.com) [:rocket:](https://synopsys.ongig.com/?l=Portugal&regions=EMEA) | Silicon chip design, verification, IP integration and application security | `Lisbon` `Porto` |
 | [Unbabel](https://unbabel.com) [:rocket:](https://unbabel.com/careers/) [:octocat:](https://github.com/Unbabel) | Translation as a Service. | `Lisboa` |
 
 
+
```
Comment (vpereira01) [inline README.md:186]:
```suggestion
```

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**